conf/cluster.yaml.sample

```diff
@@ -11,9 +11,9 @@ cluster:
       - name: hare
       - name: s3
         services:
-        - io
-        - auth
-        - bg_consumer
+        - s3server, haproxy
```

Comment:
You cannot have comma separated values... Please add new entry into the list

Reply:
sure. will update this .

Reply:
done.